### PR TITLE
Update Get-CsOnlineUser.md - added some Renamed attributes + deprecated attributes

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -234,6 +239,13 @@ The following updates are applicable for TeamsOnly customers using Microsoft Tea
 - DirSyncEnabled renamed to UserDirSyncEnabled
 - MCOValidationErrors renamed to UserValidationErrors
 - OnPremSIPEnabled renamed to OnPremIsSipEnabled
+- Enabled renamed to IsSipEnabled
+- ObjectId renamed to Identity
+- OnPremOptionFlags renamed to OnPremOptionFlag
+- OptionFlags renamed to OptionFlag
+- ProxyAddresses renamed to ProxyAddresse
```
[the remark this answers is not in the document]
Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineUser.md]: Correct, thanks for pointing this out

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by isrumnon** [comment]
Hi @get-itips @dariomws @tseward can we get this reviewed asap? Need to address some customer queries through this documentaion

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -173,6 +173,11 @@ The following updates are applicable for TeamsOnly customers using Microsoft Tea
 - Office
 - Phone
 - WindowsEmailAddress*
+- TeamsEnhancedEncryptionPolicy
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:176]: Hello! are these really not relevant to Teams? or do they belong to the filters list? Thanks for clarifying!

**Comment by isrumnon** [comment]
@get-itips let me just double check with team and update

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -230,10 +235,18 @@ The following updates are applicable for TeamsOnly customers using Microsoft Tea
...
 - DirSyncEnabled renamed to UserDirSyncEnabled
 - MCOValidationErrors renamed to UserValidationErrors
 - OnPremSIPEnabled renamed to OnPremIsSipEnabled
+- Enabled renamed to IsSipEnabled
+- ObjectId renamed to Identity
+- OnPremOptionFlags renamed to OnPremOptionFlag
+- OptionFlags renamed to OptionFlag
+- ProxyAddresses renamed to ProxyAddress
+- ShadowProxyAddresses renamed to ShadowProxyAddress
+- TeamsBranchSurvivabilityPolicy renamed to TeamsSurvivableBranchAppliancePolicy
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:249]:
```suggestion
- ObjectId renamed to Identity
- FirstName renamed to GivenName
- DirSyncEnabled renamed to UserDirSyncEnabled
- MCOValidationErrors renamed to UserValidationErrors
- OnPremSIPEnabled renamed to OnPremIsSipEnabled
- Enabled renamed to IsSipEnabled
- OnPremOptionFlags renamed to OnPremOptionFlag
- OptionFlags renamed to OptionFlag
- ProxyAddresses renamed to ProxyAddress
- ShadowProxyAddresses renamed to ShadowProxyAddress
- TeamsBranchSurvivabilityPolicy renamed to TeamsSurvivableBranchAppliancePolicy
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by isrumnon** [comment]
@dariomws thanks for pointing out that discrepancy. I've made necessary changes

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by isrumnon** [comment]
Hi @yogkumgit @tiburd can we get this merged?

**Pull request merged**